assignment7/src/App.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from "react";
+import Header from "./components/Header";
+import Search from "./components/Search";
+import PageNotFound from "./components/DeveloperCards/PageNotFound";
+import DeveloperCardsContainer from "./components/DeveloperCards/DeveloperCardsContainer";
+import Footer from "./components/Footer";
+import data from "./utils/data/data.json";
+import { Route, Routes } from "react-router-dom";
+import DeveloperDetails from "./components/DeveloperCards/DeveloperDetails";
+import FollowUp from "./components/DeveloperCards/FollowUp";
+
+const App = () => {
+  const [allDevelopersData, setAllDevelopersData] = useState([]);
+  const [filteredDeveloperData, setFilteredDeveloperData] = useState([]);
+  const [isSearched, setIsSearched] = useState(false);
+
+  //
+  useEffect(() => {
+    getDevelopers();
+  }, []);
+
+  // function of geting data of my bootcamp's developers.
+  async function getDevelopers() {
+    const developerArr = [];
+    for (developer of data) {
+      const developerDataResp = await fetch(
+        `https://api.github.com/users/${developer.github_login}`
+      );
+      const developerData = await developerDataResp.json();
+      developerArr.push(developerData);
+    }
+    setAllDevelopersData(developerArr);
+  }
+  const HomePage = function () {
```

Comment:
Add new lines between your functions and component definitions for better readability
```suggestion
  }
  
  const HomePage = function () {
```

You can also rename the HomePage component to something like MemberList and move it out into a separate file to reduce clutter